package.json

```diff
@@ -15,7 +15,7 @@
     "clean": "rm -rf ./dist",
     "serve": "yarn run build && serve .",
     "update": "npm version patch -m 'Upgrade to %s' && git push && git push --tags",
-    "deploy": "yarn build && firebase deploy && git push"
+    "deploy": "git push && yarn build && firebase deploy"
```

Comment:
Yes... but currently this is only used as a shortcut so it can be assumed the changes are committed before calling this deploy command. I can update it in the future if needed.